lib/kuzzle/internalIndexHandler.js

```diff
@@ -202,24 +203,24 @@ class InternalIndexHandler extends Store {
     await Bluebird.all(promises);
   }
 
-  async getSecret() {
-    const response = await this.get("config", this._JWT_SECRET_ID);
+  async _initSecret() {
+    const { authToken, jwt } = global.kuzzle.config.security;
+    const configSeed =
+      authToken && authToken.secret ? authToken.secret : jwt && jwt.secret;
```

Comment:
Could be simplified in `authToken?.secret ?? jwt?.secret`

Reply:
applied !